service/src/main/webapp/error/500.jsp

```diff
@@ -0,0 +1,17 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8"
+    pageEncoding="UTF-8" import="java.util.*"%>
+<html>
+    <head>
+        <title>Feckless Weasel - Server Error</title>
+        <link href="${pageContext.request.contextPath}/assets/css/base.css" rel="stylesheet" type="text/css">
+    </head>
+    <body>
+        <jsp:include page="/header.jsp"/>
+        <div class="column-beta">
```

Comment:
I don't know if this is the correct style to use for this. I might make a new style for single column pages such as this one. For now, this is fine.